src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.model;
+
+import core.basesyntax.ColorSupplier;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle() {
+        this.radius = Figure.getRandomNumber();
+        super.setColour(ColorSupplier.getRandomColor());
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-static-methods-in-your-solution)